src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/grid_options.tsx

```diff
@@ -59,6 +70,7 @@ function GridOptions({ stateParams, setValue }: VisOptionsProps<BasicVislibParam
       </EuiTitle>
 
       <SwitchOption
+        disabled={hasHistogramAgg}
```

Comment:
Is it then also possible to add a tooltip saying such? Like `tip={hasHistogramAgg ? 'X axis lines can't show for histograms' : undefined}`

Reply:
Sure. Added.

![image](https://user-images.githubusercontent.com/11524637/63003939-f9a66100-be81-11e9-9f39-955d5a02a87c.png)